neuronsimulator/templates/home.html

```diff
@@ -1,23 +1,251 @@
 {% extends "base.html" %}
 {% load static %}
+{% block title %}Neuron Simulator{% endblock %}
+
+{% block head_extras %}
+    {{ block.super }}
+    <style>
+        #div1 {
+            background-color: #f0ebcccb;
+        }
+    </style>
+
+    <!--form initial values-->
+    {{ form_init_dict|json_script:"form_init_dict" }}
+
+    <script>
+        // Initialize Bootstrap Tooltips
+        var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
+        var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
+            return new bootstrap.Tooltip(tooltipTriggerEl)
+        })
+
+        function showHideFormFields() {
+            //get init values for form parameters
+            const form_init_dict = JSON.parse(document.getElementById('form_init_dict').textContent);
+            const duration_init_value = form_init_dict["duration"]
+            const frequency_init_value = form_init_dict["frequency"]
+            const phi0_init_value = form_init_dict["phi0"]
+            const tau1_init_value = form_init_dict["tau1"]
+            const tau2_init_value = form_init_dict["tau2"]
+
+            // get value for stim_type
+            let stim_type_value = document.getElementById("id_stim_type").value;
+            let sel_option = document.getElementById("id_stim_type").selectedOptions;
+
+            // show or hide form fields based on stim_type, use inital values for fields with display=none
+            if ( stim_type_value  === 'rectangular') {
+                document.getElementById("div_duration").style.display='block';
```

Comment:
Consider using constants for the element ids as well, for the same reason as the `stim_type_value`. The values could be defined in python and inserted into the HTML (and/or JavaScript) templates, *e.g.*:

`forms.py` (Python): 
```python
DIV_DURATION="div_duration"
```
`home.html` (HTML Template): 
```html
<div id="{{ DIV_DURATION }}"
```
`home.html` (JavaScript): 
```javascript
document.getElementById("{{ DIV_DURATION }").style.display='block';
```

Reply:
see my comments above.